db/app.js

```diff
@@ -1,7 +1,37 @@
 const express = require("express");
 const app = express();
 app.use(express.json());
-const getApi = require("../db/seeds/api.controller");
+const { getApi, getTopics } = require("../db/seeds/api.controller");
 console.log("I am in app");
+
 app.get("/api", getApi);
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (req, res) => {
+  res.status(404).send({ msg: "Not Found" });
+});
+
+app.use((err, req, res, next) => {
```

Comment:
This entire block will not fire at the moment as you have nothing talking to it. Refer back to the error handling notes for full step by step on how to do this in a concise manner.